# Research Plan for Gilbert

## 1. Background training

## 1.1. SCG4 account set up
Alex Chekholko <chekh@stanford.edu>

- [x] SUnetID
- [x] dual authentication
- [x] vpn (https://uit.stanford.edu/service/vpn/)
- [x] scg4

## 1.2. Jupyter Python
- [x] kmer counting in E.coli (<a href="K-mer Counting.ipynb">link</a>) Gilbert
- [ ] [matplotlib](http://matplotlib.org/users/pyplot_tutorial.html)
- [ ] [Markdown tutorial](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)


## 1.3. Literatures
- [ ] [BSMAP: whole genome bisulfite sequence MAPping program](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-232)
- [ ] [MOABS: model based analysis of bisulfite sequencing data](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-2-r38)


# 2. Theory

## 2.1. Entropy
* input : E.coli genome, K-mer
* output : Entropy given K-mer and position
  * graph1 : position (x-axis) vs. entropy (y-axis) (K=10)
  * graph2 : histogram of entropy given K=10, x-axis is entropy.
  * graph3 : average of enropy for K=4..28  

## 2.2. DMR

